community-operators/descheduler/descheduler.v0.0.1.clusterserviceversion.yaml

```diff
@@ -159,7 +127,7 @@ spec:
                 - name: WATCH_NAMESPACE
                   valueFrom:
                     fieldRef:
-                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                      fieldPath: metadata.namespace
```

Comment:
If I am not mistaken, this confines the operator to watch own namespace only. we can no longer use an `OperatorGroup` with the operator. Is that fine?

Reply:
What I am trying to say is that you can still keep `metadata.annotations['olm.targetNamespaces']` and watch own or single namespace only.

Reply:
This shouldn't be changed, according to the OLM design for installing an operator. You can ensure that the operator is only installed into a singleNamespace OperatorGroup that watches it's own namespace by setting SingleNamespace and OwnNamespace installModes to true, which is already set.

Reply:
I'm pushing for this annotation because I would like descheduler operator's CSV to be a good example, and operators should always use the OLM annotation where applicable.

Reply:
So, you mean with the change https://github.com/operator-framework/community-operators/pull/248/files#diff-c382f070e758921b20957adf2934afe3R97, we should be good and it doesn't matter even if we have `metadata.annotations['olm.targetNamespaces`] set? If that's the case, I am fine dropping your revert commit and have only https://github.com/operator-framework/community-operators/pull/248/files#diff-c382f070e758921b20957adf2934afe3R97

Reply:
Exactly that. The annotation will always be set to the namespace OLM installs the operator into in the SingleNamespace OwnNamespace case